lib/data_migrations/in_progress_form_initial_expiration.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module DataMigrations
+  module InProgressFormInitialExpiration
+    module_function
+
+    def run
+      rows_affected = 0
+
+      InProgressForm.where(expires_at: nil).find_each do |form|
```

Comment:
`find_each` uses `find_in_batches` under the hood, so this may or may not be more efficient but I think you could do:
```ruby
InProgressForm.where(expires_at: nil).find_in_batches do |batch|
  ids = batch.collect &:id
  InProgressForm.where(id: ids).update_all(expires_at: Time.current + 60.days)
  rows_affected += batch.count
end
```
Maybe try a test locally, or if devops are fine with `find_each` then choose to ignore me 😄

Reply:
I think `find_each` will do batches of a thousand by default. find_in_batches will just pass in the array so you can walk through it that way. Not sure if one is more efficient than the other. But I can change it.